node/src/runtime/mainnet.rs

```diff
@@ -78,9 +82,30 @@ pub fn testnet_genesis(
 			max_validator_count: Some(100),
 			invulnerables: vec![],
 			slash_reward_fraction: sp_runtime::Perbill::from_percent(10),
-			stakers: initial_authorities
+			stakers: endowed_accounts
 				.iter()
-				.map(|x| (x.id.clone(), x.id.clone(), STASH, StakerStatus::Validator))
+				.map(|user| {
+					let status = if initial_authorities
+						.iter()
+						.any(|validator| validator.id == *user)
+					{
+						StakerStatus::Validator
+					} else {
+						use rand::{seq::SliceRandom, Rng};
```

Comment:
```suggestion
						use rand::{SeedableRng, seq::SliceRandom, Rng};
						let mut rng = rand::rngs::StdRng::seed_from_u64(0);
```

Might as well declare `rng` here because this appears to be where it is used...

Reply:
Yeah, but it will create rng for each invocation and produce the same results for each nominator (cause we are seeding it.)

Reply:
Okay, I understand now, n/m